core/kazoo_schemas/src/kz_json_schema.erl

```diff
@@ -41,19 +41,26 @@
 {'schema_loader_fun', fun((kz_term:ne_binary()) -> {'ok', kz_json:object()} | kz_json:object() | 'not_found')} |
 {'extra_validator', extra_validator()} |
 {'setter_fun', fun((kz_json:path(), kz_json:json_term(), kz_json:object()) -> kz_json:object())} |
-{'validator_options', validator_options()}.
+{'validator_options', validator_options()} |
+{'extra_validator_options', kz_term:proplist()}.
```

Comment:
Are there other options besides stability_level?

I'd like them enumerated here:

```erlang
-type extra_validator_option() :: {'stability_level', kz_term:ne_binary()}.
-type extra_validator_options() :: [extra_validator_option()].
```

This can be expanded as we go as well.